msteams-platform/m365-apps/publish.md

```diff
@@ -12,14 +12,15 @@ ms.subservice: m365apps
 
 Microsoft Teams supports Microsoft 365-enabled Teams apps for production. You can distribute these apps to users who use:
 
-* *Targeted Release* (dev preview) versions of Outlook.com and microsoft365.com (formerly office.com)
-* *Beta Channels* build of Outlook for Windows desktop, Outlook for Android, and Microsoft 365 Current Channel (dev preview) build of Microsoft 365 app for Android
+* Web: *Current Channel* versions of outlook.com and microsoft365.com (formerly office.com)
```

Comment:
Correction based on latest shiproom status:

* Web: *Current Channel* version of microsoft365.com (formerly office.com) and *Targeted Release* version of outlook.com